internal/olm/operator/helpers.go

```diff
@@ -34,14 +34,22 @@ func CatalogNameForPackage(pkg string) string {
 }
 
 // LoadBundle returns metadata and manifests from within bundleImage.
-func LoadBundle(ctx context.Context, bundleImage string, skipTLS bool) (registryutil.Labels, *apimanifests.Bundle, error) {
-	bundlePath, err := registryutil.ExtractBundleImage(ctx, nil, bundleImage, false, skipTLS)
-	if err != nil {
-		return nil, nil, fmt.Errorf("pull bundle image: %v", err)
+func LoadBundle(ctx context.Context, bundleRef string, skipTLS bool) (registryutil.Labels, *apimanifests.Bundle, error) {
+
+	var bundlePath string
+	if _, err := os.Stat(bundleRef); err != nil {
```

Comment:
A bit confused here. Why are we extracting contents of `bundleRef` if err is non-nil on line 40?